ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -274,6 +274,35 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Content Sharing Is Disabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    Starting with macOS 10.15, Apple has provided a control which permits a user to share
+    Apple downloaded content on all Apple devices that are signed in with the same Apple ID.
+    This allows users to share downloaded Movies, Music, or TV shows with other
+    controlled macOS, iOS and iPadOS devices, as well as photos with Apple TVs.
+    Disabling Media Sharing reduces the remote attack surface of the system
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set Media Sharing to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist WHERE
+        path LIKE '/Users/%/Library/Preferences/com.apple.amp.mediasharingd.plist' AND
+        key LIKE '%-sharing-enabled' AND
+        value = '1'
+    );
```

Comment:
From the check:
```
Note: Since the profile method sets a system-wide setting and not a user-level one,
the profile method is the preferred method. It is always better to set system-wide
than per user.
```

The profile added to this PR sets the settings system-wide. Thus the query should check that the system setting is set in `managed_policies`.

The `plist` query here would be compatible with the user level remediation script method (that uses `sudo -u <username> default write ...`).

Reply:
Double checking: Are you sticking to the profile method? If so, we should use the `managed_policies` table to check.

Reply:
I thought I had pushed a script, I'll add one once I finish 6.3.1.
I could also check for both a policy and a setting value